model/app.py

```diff
@@ -1,5 +1,47 @@
-import sys
+import json
+import os
 
+import rootpath
 
-def handler(event, context):
-    return "Hello from AWS Lambda using Python" + sys.version + "!"
+from horizon_net.horizonnet_reconstruction import HorizonNet
+import horizon_net.util as util
+
+os.chdir(rootpath.append()[-1])
```

Comment:
What is this doing out of interest?

Reply:
So rootpath is a module to make it easier the relative imports of the module located in different folders, I included it in the code cause otherwise, I was not able to import the from horizon_net folder. I realized that when fixing the flake8 issues and imports these lines got in the wrong order so I have just updated it!